### ISSUE TITLE
Add conformance badges to Readme

### ISSUE THREAD

**Issue by markapruett** [aws/eks-distro#1421]
**What would you like to be added**:
Add conformance badges to Readme [like EKS Anywhere](https://github.com/aws/eks-anywhere/)
**Why is this needed**:
Show's that we're upstream certified as a distribution

**Comment by lunarwhite** [comment]
Hi, I'd like to help this issue.

Per [EKS Anywhere](https://github.com/aws/eks-anywhere/), I think we should add something like these in the README file:

```markdown
[<img src="docs/static/images/certified-kubernetes-1.24-color.svg" height=150>](https://github.com/cncf/k8s-conformance/pull/2252)
<!--
Source: https://github.com/cncf/artwork/tree/master/projects/kubernetes/certified-kubernetes
-->
```

I also want to confirm:

- In which path should the svg file be placed? `docs/static/images/` like EKS Anywhere?
- Which certification version or versions should be added? `1.21` - `1.24`?
- Where should the content be added to the README file? [L37](https://github.com/aws/eks-distro/blob/c6ff509344841189eda83dc2b561c1ba80026f22/README.md?plain=1#L37) is OK?

Thanks in advance.

**Comment by kschumy** [comment]
@lunarwhite 

Thank you for your interest! These are all great questions.

That line looks correct, and the PR you linked is the correct one for 1.24.

* In which path should the svg file be placed? docs/static/images/ like EKS Anywhere? 
  * Let's do `docs/contents`
* Which certification version or versions should be added? 1.21 - 1.24? 
  * Just 1.24
* Where should the content be added to the README file? [L37](https://github.com/aws/eks-distro/blob/c6ff509344841189eda83dc2b561c1ba80026f22/README.md?plain=1#L37) is OK? 
  * Maybe [L40](https://github.com/aws/eks-distro/blob/c6ff509344841189eda83dc2b561c1ba80026f22/README.md?plain=1#L40) (with an extra empty line below it)

Please let us know if you have any other questions.

**Comment by lunarwhite** [comment]
I've found that the `certified Kubernetes badge` for 1.24 or above is not available yet, see this issue: https://github.com/cncf/artwork/issues/368. 

Should we temporarily use the [versionless](https://github.com/cncf/artwork/blob/master/projects/kubernetes/certified-kubernetes/versionless/color/certified-kubernetes-color.svg) badge, or just wait until it available?

**Comment by lunarwhite** [comment]
I've open a PR, thanks for your guidance.

**Comment by TerryHowe** [comment]
I think @mitalipaygude may be creating badges for EKS-A

**Comment by mitalipaygude** [comment]
Hey @TerryHowe, I saw the issue they had open on artwork repo and Taylor from cncf had a comment saying she's working on getting those out, soar the moment I'm waiting on those too.

**Comment by lunarwhite** [comment]
As this issue was solved in #1492 , we could close it.

**Comment by markapruett** [comment]
Thanks @lunarwhite!
